### PR TITLE
The Supernova application and a simple .tavis.yml file is added

### PR DESCRIPTION
**Pull request by LiaoWenyun**

### CODE REVIEW

File: SuperNova/README [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Convert the ipynb file to python script before lint:
+
+$ jupyter nbconvert SuperNova.ipynb --stdout --to python --PythonExporter.exclude_markdown=True|flake8 - --max-line-length 120 --ignore=F841,W391
```
Comment (andamian) [inline SuperNova/README:3]: We prefer to stick to 80 max line length. Where does it exceed it? In the notebook?

**Review by andamian — COMMENTED** [review]
Please add badges to the README of the repo with the python version number, build status etc. That makes it easier to navigate to the Travis status page.

File: SuperNova/README (hunk 1 above)
Comment (LiaoWenyun) [reply SuperNova/README:3]: many lines have more than 100 characters, I tried to fix it,  but the fixed version looks super ugly, do you mind?

Comment (andamian) [reply SuperNova/README:3]: https://pep8.org/#maximum-line-length
We, as a group, have resisted the urge to increase this limit. With some exceptions, there are ways to make the code look nice with the given 80 characters: use temporary variables, break chained commands, etc. If the standard library adheres to this we surely can too.

**Review by andamian — APPROVED** [review]

**Pull request merged**